frontend/src/pages/surveys/[id].tsx

```diff
@@ -0,0 +1,95 @@
+import { Container, Content, Layout } from 'styles/main.styles';
+import { Box, Button, Heading, Spinner, Tooltip } from '@chakra-ui/react';
+import Head from 'next/head';
+import { Sidemenu } from '@components/sidemenu';
+import { useRouter } from 'next/router';
+import { SurveyService } from '@api/services/survey';
+import { httpClient } from '@api/client';
+import { useEffect, useState } from 'react';
+import { Survey } from 'api/models/survey';
+import { GetServerSideProps } from 'next';
+import { AUTH_COOKIE } from 'consts';
+import { parseCookies } from 'nookies';
+import { HiArrowLeft } from 'react-icons/hi';
+import { SurveyHandler } from '@components/survey-handler';
+
+const surveyService = new SurveyService(httpClient);
+
+const SurveyPage = () => {
+  const router = useRouter();
+  const { id } = router.query;
```

Comment:
I suggest to add a validation on the `id` to prevent the user of type any value in the URL, like a word instead of a number and breaking the request